moduleconfig/src/main/java/org/jboss/pnc/common/json/moduleconfig/AlignmentConfig.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014-2019 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.common.json.moduleconfig;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.jboss.pnc.common.json.AbstractModuleConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Map;
+
+public class AlignmentConfig extends AbstractModuleConfig {
+
+    private final static Logger log = LoggerFactory.getLogger(AlignmentConfig.class);
+
+//    private List<String> mavenParameters;
+//
+//    private List<String> npmParameters;
+//
+//    private List<String> gradleParameters;
+
+    private Map<String, List<String>> alignmentParameters;
+
+    public AlignmentConfig(
+//            @JsonProperty("mavenParameters") List<String> mavenParameters,
+//            @JsonProperty("npmParameters") List<String> npmParameters,
+//            @JsonProperty("gradleParameters") List<String> gradleParameters
+            @JsonProperty("alignmentParameters") Map<String, List<String>> alignmentParameters) {
```

Comment:
We store the generic parameters as a single String in the DB and also will provide it as one String in the UI.
I think is should be just a Map<String, String>.